controllers/subscription_controller.go

```diff
@@ -47,8 +48,10 @@ type SubscriptionReconciler struct {
 	client.Client
 	Scheme            *runtime.Scheme
 	Recorder          *EventReporter
+	OperatorNamespace string
 	ConditionName     string
 	OperatorCondition conditions.Condition
+	ctx               context.Context
```

Comment:
See previous comment of why this is a bad practice and should be avoided

Reply:
removed it.